src/main/java/hw9/MultilayerStructure.java

```diff
@@ -0,0 +1,43 @@
+package hw9;
+
+import java.beans.XMLEncoder;
+import java.io.BufferedOutputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.logging.Logger;
+import java.util.regex.Matcher;
+
+public class MultilayerStructure {
+
+    private final static Logger log = Logger.getLogger(ValidateUtil.class.getName());
```

Comment:
Naming is used in the not right style. 
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names
So it should be LOG or LOGGER